delta-lake/src/integration/java/org/apache/iceberg/delta/DeltaLakeToIcebergMigrationSparkIntegration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.delta;
+
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.spark.Spark3Util;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.connector.catalog.CatalogPlugin;
+
+/** An example class shows how to use the delta lake migration actions in SparkContext. */
+class DeltaLakeToIcebergMigrationSparkIntegration {
+
+  private DeltaLakeToIcebergMigrationSparkIntegration() {}
+
+  static SnapshotDeltaLakeTable snapshotDeltaLakeTable(
```

Comment:
nit: can add a javadoc. Also `newTableIdentifier` can be both 2 parts and 3 parts identifier, is that right?

Reply:
Yes. In spark context, `Spark3Util.catalogAndIdentifier` will use default `spark_catalog` to handle the 2 parts identifier.